etchash.go

```diff
@@ -533,7 +533,7 @@ func (l *Light) getCache(blockNum uint64) *cache {
 	return c
 }
 
-/// dataset wraps an etchash dataset with some metadata to allow easier concurrent use.
+// / dataset wraps an etchash dataset with some metadata to allow easier concurrent use.
```

Comment:
Probably changed by a linter or beautifier.

```suggestion
// dataset wraps an etchash dataset with some metadata to allow easier concurrent use.
```